packages/documentation-v7/src/stories/components/collapsible/collapsible.demo.stories.ts

```diff
@@ -0,0 +1,143 @@
+
+const meta: Meta<HTMLPostCollapsibleElement> = {
+  title: 'Hidden/demos/components/Collapsible',
+  component: 'post-collapsible',
+  parameters: {
+    controls: {
+      exclude: ['Content'],
+    },
+    badges: [BADGE.BETA, BADGE.NEEDS_REVISION],
+  },
+  args: {
+    innerHTML: `<span slot="header">Titulum</span><p>Contentus momentus vero siteos et accusam iretea et justo.</p>`,
+  },
+  argTypes: {
+    innerHTML: {
+      description:
+        'Defines the HTML markup contained in the collapsible.<br/>' +
+        'Elements with a `slot="header"` attribute are displayed in the header while others are shown in the body.',
+      table: {
+        category: 'content',
+        type: {
+          summary: 'string',
+        },
+      },
+    },
+  },
+  render: (args, context) => defaultRender(args, context),
+};
+
+export default meta;
+
+type Story = StoryObj<HTMLPostCollapsibleElement>;
+
+function defaultRender(
+  args: HTMLPostCollapsibleElement,
+  context: StoryContext<HTMLPostCollapsibleElement>,
+) {
+  const hasHeader = args.innerHTML.indexOf('slot="header"') > -1;
+  const collapsibleId = `collapsible-example--${context.name.replace(/ /g, '-').toLowerCase()}`;
+
+  const collapsibleProperties = definedProperties({
+    'collapsed': args.collapsed,
+    'heading-level': args.headingLevel,
+    'id': hasHeader ? undefined : collapsibleId,
+  });
+
+  const collapsibleComponent = html`
+    <post-collapsible ${spread(collapsibleProperties)}>
+      ${unsafeHTML(args.innerHTML)}
+    </post-collapsible>
+  `;
+
+  const [currentArgs, updateArgs] = useArgs();
+
+  const toggleCollapse = (open?: boolean) => {
+    const collapsible = document.querySelector(`#${collapsibleId}`) as HTMLPostCollapsibleElement;
+    collapsible.toggle(open).then((isOpen: boolean) => {
+      console.log('currentArgs', currentArgs);
+      if (typeof currentArgs.collapsed !== 'undefined') updateArgs({ collapsed: !isOpen });
+    });
+  };
+
+  const togglers = [
+    ['Toggle', () => toggleCollapse()],
+    ['Show', () => toggleCollapse(true)],
+    ['Hide', () => toggleCollapse(false)],
+  ];
+
+  const togglersHtml = html`
+    <div class="d-flex gap-mini mb-regular">
+      ${togglers.map(
+        ([label, listener]) =>
+          html`
+            <button
+              aria-controls="${collapsibleId}"
+              aria-expanded="${!args.collapsed}"
+              class="btn btn-secondary"
+              @click="${listener}"
+              key="${label}"
+            >
+              ${label}
+            </button>
+          `,
+      )}
+    </div>
+  `;
+
+  return html`
+    ${hasHeader ? null : togglersHtml} ${collapsibleComponent}
+  `;
+}
+
+export const Default: Story = {};
+
+export const InitiallyCollapsed: Story = {
+  parameters: {
+    controls: {
+      exclude: ['heading-level', 'toggle'],
+    },
+  },
+  args: { collapsed: true },
+};
+
+export const HeadingLevel: Story = {
+  parameters: {

```

Comment:
No that the stories are hidden, you can potentially remove the `parameters` to simplify the file.